src/content/user-guide/deepsparse-engine/deepsparse-enterprise.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: "DeepSparse Enterprise"
+metaTitle: "DeepSparse Enterprise"
+metaDescription: "Deepsparse Enterprise"
+githubURL: "https://github.com/neuralmagic/docs/blob/main/src/content/user-guide/deepsparse-engine/deepsparse-enterprise.mdx"
+index: 6000
+---
+
+# DeepSparse Enterprise Edition
+
+The DeepSparse Engine is available in two editions: 
+1. **The Community Edition** is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).
+2. **The Enterprise Edition** requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications. 
+
+## Installation
+Install the Enterprise Edition as follows:
+
+```bash
+pip install deepsparse-ent
+```
+
+### Getting a License
+The DeepSparse Enterprise Edition requires a valid license to run the engine and can be licensed for production, commercial applications. There are two options available:
+
+#### 90-Day Enterprise Trial License
+To try out the DeepSparse Enterprise Edition and get a Neural Magic Trial License, complete our [registration form](https://neuralmagic.com/deepsparse-engine-free-trial). Upon submission, the license will be emailed immediately to you and your 90-day term starts right then.
```

Comment:
@rsnm2 can we just cut the word immediately here, don't want to give the user a false expectation (it will come within two minutes, I missed this on the GH one too and updated there as well).